test/unit/recipes/ohai_plugin_spec.rb

```diff
@@ -22,28 +22,11 @@
 describe 'dovecot::ohai_plugin' do
   let(:chef_run) { ChefSpec::SoloRunner.new.converge(described_recipe) }
 
-  it 'creates ohai reload resource' do
-    resource = chef_run.ohai('reload_dovecot')
-    expect(resource).to do_nothing
-  end
-
-  it 'installs dovecot plugin' do
-    expect(chef_run).to create_template('/etc/chef/ohai_plugins/dovecot.rb')
-      .with_owner('root')
-      .with_group('root')
-      .with_mode('0755')
-  end
-
-  it 'dovecot plugin installation notifies ohai reload' do
-    resource = chef_run.template('/etc/chef/ohai_plugins/dovecot.rb')
-    expect(resource).to notify('ohai[reload_dovecot]').to(:reload).immediately
-  end
```

Comment:
I think all these tests should be replaced by a [`create_ohai_plugin` matcher](https://github.com/chef-cookbooks/ohai#chefspec-matchers-1). Something like the following:

```ruby
expect(chef_run).to create_ohai_plugin('dovecot')
```